doc/make/index.md

```diff
@@ -88,6 +88,10 @@ Pass these along with the `cmake ../../src` command.
   every `git commit`. Use this option to avoid the version check. The `.olean`
   files can be removed manually by invoking `make/ninja clean-olean`.
 
+Incremental Builds
+------------------
+To speed up interactive development, you can use the `shell/bin_lean` Make/Ninja target, which will build the Lean executable (into `bin/`), but not all the tests.
```

Comment:
Can you give the example of the command line? Is it `ninja shell/bin_lean`?

Reply:
I added explicit commands.